tests/Unit/Order/FeedTest.php

```diff
@@ -0,0 +1,42 @@
+<?php
+namespace BazaarVoice\Order;
+
+use BazaarVoice\Elements\InteractionElement;
+use BazaarVoice\Elements\ProductElement;
+use PHPUnit\Framework\TestCase;
+
+class FeedTest extends TestCase
+{
+    public function testNewOrderElement()
+    {
+        // Set
+        $feed = new Feed();
+        $transationDate = '2018-07-11T08:36:47';
+        $emailAddress = 'john@example.com';
+        $userName = 'John Doe';
+        $userId = substr(md5(uniqid()), 0, 8);
+        $locale = 'pt_BR';
+        $products = [
+            new ProductElement(
+                substr(md5(uniqid()), 0, 8),
```

Comment:
There is a problem to hard code this? Generating a hash just make the test longer to run.